### DISCUSSION TITLE
Sixel not working for wsl?

### DISCUSSION THREAD

**Discussion by benhaotang** [Eugeny/tabby#6987] [Q&A]
I have enabled sixel support in settings, but lsix and viu in ubuntu-wsl2 still reports that "Error: Your terminal does not report having sixel graphics support."
![](https://user-images.githubusercontent.com/50174724/184810831-0d528af9-33c8-4f9e-b8ca-e5d610c4fc1f.png)
I am using tabby 183 on win11 with ubuntu-wsl2, can confirm that lsix work on x11 xterm and wsltty.
I also can confirm that through ssh, sixel works for tabby on remote sessions.
how to solve this? is there any setting that I have done wrong?

**Comment by ysc3839** [comment]
You can try `export LSIX_FORCE_SIXEL_SUPPORT=1`

**Reply by benhaotang** [reply]
well, I think it is not this case, aside from lsix, the sixel mode for w3m, euporie ,etc, all don't work, just print an empty line. I think my entire wsl shell just cannot use the sixel inside of tabby.

**Reply by ysc3839** [reply]
I think it's Windows console host's fault. Currently Tabby uses conhost as a middle layer. The conhost works like screen/tmux, the command line program output to conhost, then conhost update it's "screen" and output changed parts of the screen to terminal. So if conhost doesn't understand something, it would just ignore.

**Reply by benhaotang** [reply]
I think this is the answer.
